packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -15,7 +16,10 @@ import {format} from './mutationFormatter'
 
 const helpText = `
 Options
-  --dryRun <boolean> Whether or not to dry run the migration. Default to true, to actually run the migration this has to be set to false
+  --dry <boolean> Whether or not to dry run the migration. Default to true, to actually run the migration this has to be set to false
```

Comment:
_Tiny thing_. This description should end with a period to be consistent with the other option descriptions.